yarn.lock

```diff
@@ -40,6 +40,17 @@
     source-map "^0.5.0"
     trim-right "^1.0.1"
 
+"@babel/generator@^7.4.0":
+  version "7.4.0"
+  resolved "http://artifactory.corp.linkedin.com:8081/artifactory/api/npm/npm-external-new/@babel/generator/-/@babel/generator-7.4.0.tgz#c230e79589ae7a729fd4631b9ded4dc220418196"
```

Comment:
I'm guessing this domain won't resolve from the outside world?

Reply:
ah lol, I guess something got messed up with my config